_addons_transformations/scale/readme.md

```diff
@@ -14,17 +14,47 @@ install: auto
 
 # Scale Transformation Service
 
-Transform the input by matching it between limits of ranges in a scale file.  The input string must be in numerical format.
+Transform a given input by matching it between limits of ranges.
+The input string must be in numerical format.
 
-The file is expected to exist in the `transform` directory, and should follow the format given in the example below.
+The file is expected to exist in the `transform` directory, and should follow the format given in the table below.
+
+| scale expression | returns XYZ when the given value is |
+|--------------------|--------|
+| `[12..23]=XYZ` | `between (or equal to) 12 and 23` |
+| `]12..23[=XYZ` | `between 12 and 23 (12 and 23 are excluded in this case.)` |
+| `..23]=XYZ` | `lower than or equal to 23` |
+| `]12..=XYZ` | `greater than 12` |
```

Comment:
Good examples :)

Reply:
We should add a sentence like "Lines are evaluated from top to bottom" - I did not test this! That's how I understood it from sourcecode.

Reply:
That would also answer my question regarding overlapping ranges

Reply:
>We should add a sentence like "Lines are evaluated from top to bottom" - I did not test this! 

Hm i don't se an effort here, since the order is irrelevant. (See last review comments.)